packages/core/http/core-http-router-server-internal/src/versioned_router/core_versioned_route.test.ts

```diff
@@ -181,66 +210,4 @@ describe('Versioned route', () => {
     expect(validatedQuery).toBe(true);
     expect(validatedOutputBody).toBe(true);
   });
-
-  it('returns the expected output for non-existent versions', async () => {
```

Comment:
Rather run these tests as part of the `jest_integration` tests since they are testing the router from the perspective of the client.